config/brakeman.ignore

```diff
@@ -974,6 +951,28 @@
       ],
       "note": ""
     },
+    {
+      "warning_type": "Cross-Site Scripting",
+      "warning_code": 2,
+      "fingerprint": "4f50bd02cffae24567d5d511f692ceeae1ac8183f4092cb176a63afabf3b18df",
+      "check_name": "CrossSiteScripting",
+      "message": "Unescaped model attribute",
+      "file": "drivers/hmis_external_apis/app/views/hmis_external_apis/external_forms/form.haml",
+      "line": 4,
+      "link": "https://brakemanscanner.org/docs/warning_types/cross_site_scripting",
+      "code": "HmisExternalApis::ExternalForms::FormGenerator.new(self, @form_definition).render_node(node)",
+      "render_path": null,
+      "location": {
+        "type": "template",
+        "template": "hmis_external_apis/external_forms/form"
+      },
```

Comment:
This was already being ignored. The change I made was to pass the `@form_definition` record to the FormGenerator. I'm not really wrapping my head around how this could be vulnerable to XSS, since nothing is coming from the browser (Except in dev mode), we are just generating this HTML using a view. So I think it is a false positive, but I would appreciate a look.